test/test_enclave_wrangler/test_objects_api.py

```diff
@@ -211,3 +233,206 @@ def test_get_concept_set_version_members(self):
         obj_id = 563193300
         data = get_concept_set_version_members(obj_id, return_detail='full')
         self.assertGreater(len(data), 0)
+
+
+# TODO: new tests by claude to try
+# from pprint import pprint as pp
+# todo: #1: Wherever I put '#1', consider removing the mock and call the actual function
+class TestObjectsApiMocks(DbRefreshTestWrapper):
+    """Tests that specifically use mock APIs
+
+    Scaffolded by: https://claude.ai/chat/7f64ea42-7ab2-43c6-9a86-382bdafc26bf
+    """
+    func_cached_return_map = {
+        # test_get_object_types()
+        get_object_types: os.path.join(TEST_INPUT_DIR, 'test_get_object_types', 'object_types.json')
+    }
+
+    def setUp(self):
+        """Set up mock connection"""
+        self.mock_connection = MagicMock()
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up inputs: cached return data for functions that use mock API calls"""
+        for func, path in cls.func_cached_return_map.items():
+            if not os.path.exists(path):
+                os.makedirs(os.path.dirname(path), exist_ok=True)
+                with open(path, 'w') as f:
+                    json.dump(func(), f)
+
+    def _read_cached_return(self, func: Callable):
+        """Read cached return from file"""
+        path = self.func_cached_return_map[func]
+        with open(path, 'r') as f:
+            return json.load(f)
+
+    # TODO: dryify this to re-use?
+    #  - one difficulty is {'data': response} and other optional small mutations various funcs implement.
+    @patch('enclave_wrangler.objects_api.enclave_get')  # todo #1
+    def test_get_object_types(self, mock_enclave_get):
+        """Test get_object_types()
```

Comment:
## Test `get_object_types()`
The first test. It's kind of mundane. But basically I make sure we get back at least these endpoints:
`'OMOPConcept', 'OMOPConceptSetContainer', 'OMOPConceptSet', 'OmopConceptSetVersionItem'`

However this test is just a mock. I think it's good to have mock tests, just to make sure that no essential basic logic around API calls gets botched. However, it would indeed be good to call the actual enclave API itself.